.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules
+**/dist
+**/.eslintrc.js
```

Comment:
Despite being ignored, `.eslintrc.js` files should still get properly formatted by the editor, assuming it has support for Prettier and EditorConfig.